lesson-16/src/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ru">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Lesson 16</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <article class="block-changing">
+    <article>
+      <a href="">
+        <p>Я содержимое которое нужно изменить</p>
+      </a>
+      <button type="button" id="change-content">Изменить содержимое</button>
+    </article>
+    <article>
+      <a href="" id="element-for-change">
+        <p>Мой родитель это элемент который нужно заменить на другой</p>
+      </a>
```

Comment:
```suggestion
      <a href="" id="element-for-change">Мой родитель это элемент который нужно заменить на другой</a>
```

текст внутри ссылки не нужно оборачивать в `<p>`

Reply:
так было задумано, чтобы посмотреть, что все содержимое заменится